CHANGELOG.md

```diff
@@ -6,6 +6,16 @@ Breaking changes are prefixed with a "[BREAKING]" label.
 ## master (unreleased)
 
 
+## 0.6.0 (2019-10-08)
+
+This release refactors the internal data structures of Rafka. The external API and generally any
```

Comment:
Let's add a more clear comment on removing the ability for multiple consumers per client.